ui/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "jupyterhub-singleuser-profiles-ui",
+  "version": "0.1.0",
+  "private": true,
+  "homepage": "http://127.0.0.1/hub/static/jsp-ui",
```

Comment:
Should this be a link to GH repo?

Reply:
It is actually used by the build: 
```The project was built assuming it is hosted at /hub/static/jsp-ui/.
You can control this with the homepage field in your package.json.```
Unless it causes problems I will leave it as it is for now so it does not cause problems